### ISSUE TITLE
Create ADR summarizing design decisions about spheres and guilds

### ISSUE THREAD

**Issue by fancysnake** [zagrajmy/adr#4]
We have two kinds of groups:
Sphere:

- has guilds inside
- is tied to a subdomain (wroclaw.rpg.zagrajmy.net) or domain (pizza.kobolds.club)
- has a general guild that is a default for all users
- there is a general sphere, where you can create guilds not tied to a single sphere
- each meeting has a guild assigned
- spheres are managed by administration. You cannot create one (they will be payed in future)

Guild:
- Every user can create it
- Guilds can have different access policies (open, invitation)

**Comment by hasparus** [comment]
> is tied to a subdomain (wroclaw.rpg.zagrajmy.net)

Or domain. `super-boardgames-con.pizza`

**Comment by hasparus** [comment]
Does every user start with it's own private guild? Kinda like `team hasparus` on Netlify or Sentry.

This would make creating non-bullshit private events possible.

**Comment by fancysnake** [comment]
No, when you register on our site, you are added to a sphere and its default guild.

**Comment by hasparus** [comment]
Okay, so if you want "private events" just create a new guild for your new group. Cool.
